app/main.dev.ts

```diff
@@ -31,12 +31,18 @@ import { registerIpcStoreHandlers } from './main/register-store-handlers';
 import { registerIpcContextMenuHandlers } from './main/register-context-menus';
 import { addMacOsTouchBarMenu } from './main/macos-touchbar-menu';
 import { registerThemeModeHandlers } from './main/register-theme-mode-handlers';
-import { registerSentryStateListener } from './main/register-sentry-state-listener';
+import {
+  initializeSentry,
+  registerSentryStateListener,
+} from './main/register-sentry-state-listener';
 
 // CSP enabled in production mode, don't warn in development
 delete process.env.ELECTRON_ENABLE_SECURITY_WARNINGS;
 process.env.ELECTRON_DISABLE_SECURITY_WARNINGS = 'true';
 
+const store = registerIpcStoreHandlers(getUserDataPath(app));
```

Comment:
You're certain the return value of `getUserDataPath(app)` is the same outside of when.ready?

Reply:
no :) I tested a few use cases and it seems to work. are you aware of some use cases we should test to confirm this?

Reply:
I vaguely recall some funky behaviour with this fn, so recommend double checking.